### ISSUE TITLE
Unable to build Docker container

### ISSUE THREAD

**Issue by fgnm** [ICCD-MiBACT/ArCo#233]
I'm unable to build the docker image downloaded from the master branch. I just run the `docker-compose build` and after a while I got this error
```
--2022-05-24 19:12:44--  http://arco.istc.cnr.it/arco-data/1.0.1/index.txt
Resolving arco.istc.cnr.it (arco.istc.cnr.it)... 150.146.207.164
Connecting to arco.istc.cnr.it (arco.istc.cnr.it)|150.146.207.164|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-05-24 19:12:44 ERROR 404: Not Found.

/download_release.sh: line 10: index.txt: No such file or directory
creating directory: 1
mv: cannot stat '*.nt.gz': No such file or directory
rm: cannot remove 'index.txt': No such file or directory
--2022-05-24 19:12:44--  http://arco.istc.cnr.it/arco-data/1.0.1/linking/index.txt
Resolving arco.istc.cnr.it (arco.istc.cnr.it)... 150.146.207.164
Connecting to arco.istc.cnr.it (arco.istc.cnr.it)|150.146.207.164|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-05-24 19:12:44 ERROR 404: Not Found.

creating directory: 1
/download_release.sh: line 29: index.txt: No such file or directory
mv: cannot stat '*.nt.gz': No such file or directory
rm: cannot remove 'index.txt': No such file or directory
--2022-05-24 19:12:44--  http://arco.istc.cnr.it/arco-data/1.0.1/quarantine.nt.gz
Resolving arco.istc.cnr.it (arco.istc.cnr.it)... 150.146.207.164
Connecting to arco.istc.cnr.it (arco.istc.cnr.it)|150.146.207.164|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2022-05-24 19:12:44 ERROR 404: Not Found.
```

Indeed, the page `http://arco.istc.cnr.it/arco-data/1.0.1/index.txt` and the others returns 404.

**Comment by luigi-asprino** [comment]
Hi @fgnm 

which version you are trying to build? 
Could you please post the command line you are executing?

**Comment by fgnm** [comment]
@luigi-asprino as I said in the first post:
- I downloaded the master branch
- I run the command `docker-compose build` (of course inside `ArCo-release` folder)

Container builds fails with the error posted above. For instance the links are broken because returns 404

**Comment by luigi-asprino** [comment]
Ok, the docker-compose.yml was set for an old version of the KG (which is not available anymore).
With [1134fea](https://github.com/ICCD-MiBACT/ArCo/commit/1134fea540882a4827332516afa23489277d7110) I updated the configuration to the latest version available (1.1).

**Comment by fgnm** [comment]
Thanks or worked
